AnnoyancesFilter/sections/popups.txt

```diff
@@ -117,6 +117,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+oceans-nadia.com###nadia-popup
```

Comment:
This can lock scrolling, change to
```
oceans-nadia.com#$##nadia-popup { display: none!important; }
oceans-nadia.com#$##popup-overlay { display: none!important; }
oceans-nadia.com#$#html[style="overflow: hidden;"] { overflow: auto!important; }
```

Reply:
ok.done